shapeshift/web
# chore: upgrade swappers to structurally support multi-hop trades

## Description

Upgrades swappers to support multi-hop trades structurally, without actually implementing multi-hop trades in the application. Also removes redundant `sendMax` and `maximumCryptoHuman` from swappers.

## Pull Request Type


## Issue (if applicable)

NA

## Risk

This is a fairly high risk change.
- Risk of trades generally broken
- Risk of broken max trades

## Testing

This change should introduce zero change from production.

- check that quotes are being created correctly including fees and receive amounts
- check that trades execute correctly for all swappers
- check that max trades execute correctly for all swappers

### Engineering

See above

### Operations

This change should introduce zero change from production.

## Screenshots (if applicable)
